boards/posix/native_posix/native_posix.dts

```diff
@@ -194,4 +194,13 @@
 		sample-point = <875>;
```

Comment:
Typo in commit msg of "drivers: can: native_posix_linux: initial implementation"
s/applicatoin/application/